Source/WebCore/rendering/style/StyleRareNonInheritedData.h

```diff
@@ -173,6 +173,8 @@ class StyleRareNonInheritedData : public RefCounted<StyleRareNonInheritedData> {
 
     float zoom;
 
+    std::optional<Length> blockStepSize;
```

Comment:
You need to compare in the equality operator as well and add it to the copy constructor.

https://searchfox.org/wubkat/source/Source/WebCore/rendering/style/StyleRareNonInheritedData.cpp#161
https://searchfox.org/wubkat/rev/6312ca8a662145d355274780bbf68b6ce735d8e8/Source/WebCore/rendering/style/StyleRareNonInheritedData.cpp#283

Reply:
Think I got everything but let me know if I missed something